docs/building.md

```diff
@@ -75,20 +75,20 @@ make out/trivialDAO_storage.tz \
   guardian_address=KT1QbdJ7M7uAQZwLpvzerUyk7LYkJWDL7eDh \
   governance_token_address=KT1RdwP8XJPjFyGoUsXFQnQo1yNm6gUqVdp5 \
   governance_token_id=0n \
-  now_val=`date +"%s"` \
+  start_level=100n \
   metadata_map=Big_map.empty \
   freeze_history=[] \
   fixed_proposal_fee_in_token=0n \
   quorum_threshold=10n \
   min_quorum=1n \
   max_quorum=99n \
   max_voters=1000n \
-  period=950400n \
+  period=15840n  \
   quorum_change=5n \
   max_quorum_change=19n \
   governance_total_supply=1000n \
-  proposal_flush_time=1900801n \
-  proposal_expired_time=2851200n
+  proposal_flush_level=36000n  \
+  proposal_expired_level=47520n \
 ```
 
 The `admin_address`, `guardian_address`, `governance_token_address`, and `governance_token_id`
```

Comment:
This should list the `start_level` to, since it's now required (same for the two variants below)

Reply:
Yes. added.